metadata.json

```diff
@@ -18,8 +18,6 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "7",
-        "8",
```

Comment:
Might be good to track these changes in an issue for the changelog.

Reply:
OK.  It doesn't look like there is an issue label for breaking changes?